app/src/main/java/com/keylesspalace/tusky/service/SendStatusService.kt

```diff
@@ -181,6 +182,20 @@ class SendStatusService : Service(), Injectable {
                 return@launch
             }
 
+            media.forEach { mediaItem ->
+                if (mediaItem.processed) {
+                    val focus = mediaItem.focus
+                    val focusString = if (focus != null) "${focus.x},${focus.y}" else null
+                    Log.w(TAG, "updating media description on status send "+mediaItem.description )
+                    mastodonApi.updateMedia(mediaItem.id!!, mediaItem.description, focusString)
+                        .fold({
+                        }, { throwable ->
+                            Toast.makeText(applicationContext, R.string.error_failed_set_caption, Toast.LENGTH_SHORT).show()
```

Comment:
The message/case now leaves out the "focus failed" message.

I guess the one here should be a more generic one (and the old two removed).

Reply:
I believe that we prefer snackbars for these

Reply:
This needs to be guarded with `!editing` like the previous block in `updateMediaItem` was, for the same reason. Testing locally, I hit the failure block here when editing the caption and focus for an attachment to a published status.

Reply:
Re snackbar: That absolutely needs a view.

However this is a service so more or less the opposite.

Other errors are reported as actual notifications here.

So I'm not sure it should even show a Toast (but it probably doesn't hurt).

Reply:
Ah right, good point @ view